frontend/app/src/entities/nodes/relationships/utils/dissociate.test.tsx

```diff
@@ -0,0 +1,78 @@
+import { getDissociateAction } from ".";
+import {
+  deviceMockSchema,
+  deviceRelationshipMinCountMockSchema,
+  deviceRelationshipNotOptionalMockSchema,
+} from "../../../../../tests/mocks/data/devices";
+import {
+  interfaceL2MockSchema,
+  interfaceL2WithoutDeviceMockSchema,
+} from "../../../../../tests/mocks/data/interfaces";
+
+describe("Dissociate action", () => {
```

Comment:
test does not need to be `.tsx`, just `.ts`.
Also the naming convention are `function-name.test.ts`